blocks/graph.py

```diff
@@ -354,3 +414,104 @@ def apply_noise(computation_graph, variables, level, seed=None):
         replace[variable] = (variable +
                              rng.normal(variable.shape, std=level))
     return computation_graph.replace(replace)
+
+
+def apply_dropout(computation_graph, variables, drop_prob=0.5, rng=None,
```

Comment:
Don't fully agree, almost everyone uses 0.5. However, there are actually cases where people use a scaling which is not directly inverse to the dropout probability, so you might want allow users to (optionally) pas that too.